### ISSUE TITLE
i have 16 TB in GUI but 0 B in Console

### ISSUE THREAD

**Issue by bashar1988** [hpool-dev/chia-miner#567]
i can see the 16 TB in GUI but in the Console there is 0 B 
![image](https://user-images.githubusercontent.com/83901628/120656780-33e05880-c484-11eb-833d-ac0213ede552.png)
any idees why thats happend 
![image](https://user-images.githubusercontent.com/83901628/120656912-4ce90980-c484-11eb-9470-1a7242c629a7.png)

**Comment by iamchriswick** [comment]
I have over 6000!!! plots now invalid with the new miner. As good a time as any to switch over to a more serious pool!

**Comment by kynhanderson** [comment]
level=error msg="add plot file" error="badbit or failbit after reading size 104 at position 0" f7="{loggers.go,logging.CPrint,156}" f8="{plotterSpace.go,chia.(*PlotterSpace).Load,240}" f9="{asm_amd64.s,runtime.goexit,1371}"

**Comment by kynhanderson** [comment]
Hey Guys, just CLOSE GUI and CONSOLE, remove LOG files and open the CONSOLE again and let it check files. and all TB will be back. Sorry english. Brazil here =x

**Comment by atasener** [comment]
same issue started ...  any solution?

**Comment by brasrox** [comment]
**ONLY** use console OR Gui ! chose one and use it

**Comment by atasener** [comment]
Yeah its work thanks.

**Comment by Bluecartwheel** [comment]
> Yeah its work thanks.

What works? LOL, idiot.

**Comment by atasener** [comment]
@Bluecartwheel  Of course, your capacity was not enough to understand this. You are most idiot here..

@brasrox commented 2 days ago;
ONLY use console OR Gui ! chose one and use it
@atasener  
Yeah its work thanks.
